packages/syft/src/syft/client/client.py

```diff
@@ -454,6 +476,25 @@ def post_init(self) -> None:
         if self.metadata is None:
             self._fetch_node_metadata(self.credentials)
 
+        self.communication_protocol = self.__get_communication_protocol(
```

Comment:
Double __ on the left hand side causes name mangling. I am not really a fan of this, considering you can still overwrite it anyway even when mangled. From what I can tell the only good use of it would be multiple inheritance to avoid conflicting two methods with the same name.

Is there another reason why you think we should use it, or could this be renamed to _get_communication_protocol?